### ISSUE TITLE
Create automation creator landing page guide

### ISSUE THREAD

**Issue by Olivia-Chiong** [100Automations/Website#130]
### Overview
We need to create a landing page for the automation creation process so users can understand the process

### Action Items
An overview of the automation creation process (with screenshots of the futureautomations board) and links to each of the 3 guides

Product
- [x] Write a draft outline
- [x] Paste mark-down for the guide instructions in a comment on this issue.
- [ ] Review with team
- [ ] Create a mark-down file for this guide and add it to a folder called _guides
   - [ ] Call the file a name with dashes that is as short as possible
      - [ ] name:
   - [ ] Add a header to the file and add the name of guide where it says title
```
---
title:
description:
related guides:
   - title:
     url:
status: coming soon
display: true
---
```

### Resources/Instructions
#125

**Comment by NivenPrasad** [comment]
---
title: Self-Evaluating A New Automation Idea
description: Research and questions for helping you move forward with your automation
status: coming soon
display: true
---

# Self-Evaluating A New Automation Idea

## Try writing a short description that explains your automation
In just a sentence or two, explain the repetitive thing you have to do, and the benefit of automating it. Your description should be clear enough so that developers will understand what it does, but not use too much jargon so less technical users can understand how it might help them. Coming up with this brief summary can help you clarify your idea, so it may take a few iterations to refine it. 

For tips on how to to this, read the guide for [writing a description of your automation](https://github.com/100Automations/Website/blob/master/_guides/writing-short-descriptions-for-automations.md)

## See if there are already tools that try to solve this problem
Check major code repositories and developer communities (GitHub, StackOverflow, NPM) to see if there are already similar tools. If there are, describe how your idea makes things easier or takes a different approach. For example would it be more efficient, or unattached to another project?

## Think of who your automation will help and what problems it solves
Describe who your automation will help and how. Also, consider what help you might need with your automation. Do you have the code and need an interface? Do you have a mock-up of your idea and need help with the code? Think about less technical needs, like writing and testing. 

## Consider the time your idea would take to develop compared to the time saved
Try to convey a sense of how much time it might take to fully develop your idea, and how much time your automation could save compared to how things are done now. Does it solve a small problem for a lot of people? A big problem for a few? 

## Start documenting your idea 
Look at the [guide for creating a good ReadMe](https://github.com/100Automations/Website/blob/master/_guides/creating-good-readmes-for-automations.md) and based on your idea, try to fill out what you know. This should help you understand what questions you'll need to answer as you develop your automation. If you've already started working on your automation, describe how you expect it to work (what triggers it, what the result will be) and what the requirements are (language, platform, dependencies).

**Comment by NivenPrasad** [comment]
Status: In progress
Blockers: None
Availability: Available
Eta: By end of Monday to leave time for review by product on Tue or Wednesday if possible and then ready for team to see on Wednesday.

**Comment by NivenPrasad** [comment]
@mdothman @ExperimentsInHonesty Q for development - will including screenshots in the overview or for that matter, guides, pose any issues from a development standpoint?

**Comment by Olivia-Chiong** [comment]
Screenshots can be used in the guides and they will be saved in /assets/images/guides/guidename

**Comment by NivenPrasad** [comment]
Completed the draft in above comment but have not had a chance to get it reviewed (have been unwell last couple of days). We also need to confirm whether this would be better in the Future Automations wiki as opposed to a Overview/Guide (too long for an Overview was a good call-out so maybe whether it should be a Guide vs in the Wiki or both).

**Comment by NivenPrasad** [comment]
This has been broken up into 2: (1) Guides for Ideators and (2) Guide for Builders to be workshopped in the Future Automations Wiki.

**Comment by Olivia-Chiong** [comment]
We need to convert this to markdown.

**Comment by NivenPrasad** [comment]
---
title: How to build an automation 
description: There are many ways to get involved in building an automation. This guide sets out how to get involved in automation building, depending on what level you are and how you'd like to get involved.
status: coming soon
display: true
---
# How to build an automation

## Want to Build an Automation?

You can help build automations that are at different phases and find opportunities that suit your level:

1. You have your own idea and you want to build it yourself
2. You want to build an automation from scratch but don’t have your own idea
3. You want to contribute to an automation that’s already being developed

See below for how to get started!

## Build Your Own Automation
So you have an idea for an automation and want to build it yourself? You can either fill out this [Automation Form](url) to get started or if you're comfortable with GitHub, get started with this [Automation proposal template](https://github.com/100Automations/futureautomations/issues/new?assignees=&labels=documentation%2C+review&template=-automation-proposal.md&title=%5BAutomation+Name%5D+Proposal).

If you use the Automation Form, one of the 100 Automations team will take the information, create an issue card for you in the automation creation workflow in GitHub, and reach out to you once that is ready.

If you use the Automation Proposal template, our [automation workflow overview](https://github.com/100Automations/futureautomations/wiki/Ideators-Guide) will help you get an understanding of how you can use this to develop automations.

When your automations get to the "In progress" phase of the workflow, be sure to read our [Guide on Creating an Automation Repo](url) to get you set up at the build phase.

## Build From Automations Log
Don't have your own idea? We have a log of well-developed and documented automation ideas vetted by the 100 Automations team that need development. So you can jump right in and start developing.

To find an automation you'd like to build, go to the [Automations Log here](https://github.com/100Automations/futureautomations/projects/1#column-9876973), self-assign yourself on the issue, and move it to the In Progress (actively working) column. 

Again, our [Guide on Creating an Automation Repo](url) will help you get up and running at this phase.

## Improve An Automation
Some automations are already in development but still need help. Click on the "needs help" part of an automation card on our home page and see what type of help is needed for that automation. To find out more about an automation, click on the Readme button. 

Here is the [anatomy of an automation card](url) to hep you navigate around automations on our homepage.

## Need help at any stage?

If you need help at any stage of the build process, be sure to move your automation card in the [Futureautomations board](https://github.com/100Automations/futureautomations/projects/1) to the [Questions for the 100 Automations Team](https://github.com/100Automations/futureautomations/projects/1#column-12391599) column.

If your automation needs help, in the form of peer review, maintenance, support for users, refactoring, or documentation, make sure to apply the "Needs help" and the type of help labels to your automation card.

Happy developing!

**Comment by NivenPrasad** [comment]
Status: In progress
Blockers: None
Availability: Available
Eta: Draft ready for review by Wednesday 02/24

**Comment by NivenPrasad** [comment]
Made some progress but got waylaid with some other work. Should have a complete draft by Saturday.

**Comment by NivenPrasad** [comment]
First draft ready for review by product. Please bullet point general comments and point out typos in bullet point format as a comment in this issue.

**Comment by Olivia-Chiong** [comment]
@akibrhast can you take a look at the screenshots here and parse it into the guide?

**Comment by henlatourrette** [comment]
I understand that there are two routes depending on whether I would submit an automation idea or I would build the automation.

For the case of submitting an idea:
- To keep the same tone as in the builder's guide, I would begin by saying "You can propose your idea through the workflow tool we have created for this purpose..." or something similar. And then add/or include the content from the [Wiki link](https://github.com/100Automations/futureautomations/wiki/Ideators-Guide) - which I think is also already included in here.

For the case of the Builder's view:
- I think it's perfect. There's the link to the [Wiki Guide for Ideators](https://github.com/100Automations/futureautomations/wiki/Ideators-Guide) included, which I think it's fine.  

Anyway, I'd like to understand better how will this work from the user's perspective (how will a user arrive to this landing page?). But those are my comments so far.

**Comment by Olivia-Chiong** [comment]
My only concern about the length of the headers and how it will translate into the table of contents. As discussed during our meeting on 3/20/21 we will use a [header for mobile](https://www.figma.com/file/zGyhV8Z6sLohatYAnFoxQm/100Automations-Project-Board?node-id=2336%3A7929)

Breakpoints discussed

/**
 * Breakpoints
 */
$screen-desktop: 960px;
$screen-tablet: 768px;
$screen-mobile: 480px;

$bp-desktop-up: '(min-width: #{$screen-desktop})';
$bp-below-desktop: '(max-width: #{$screen-desktop - 1})';
$bp-tablet-up: '(min-width: #{$screen-tablet})';
$bp-below-tablet: '(max-width: #{$screen-tablet - 1})';
$bp-mobile-up: '(min-width: #{$screen-mobile})';
$bp-below-mobile: '(max-width: #{$screen-mobile - 1})';

/**

**Comment by NivenPrasad** [comment]
Next step: separate Builder's Guide which [has already started in the above thread](https://github.com/100Automations/Website/issues/130#issuecomment-782747714) - make a file in the _guides folder

**Comment by NivenPrasad** [comment]
Status: In progress
Blockers: Working this week
Availability: Available
Eta: By Saturday Apr 24

**Comment by NivenPrasad** [comment]
See linked pull request. This should also fix #129
